src/victory-primitives/candle.js

```diff
@@ -74,30 +77,50 @@ export default class Candle extends React.Component {
   }
 
   getCandleProps(props) {
-    const { candleHeight, x, y, events, role, className } = props;
+    const { candleHeight, x, open, close, events, role, className } = props;
     const shapeRendering = props.shapeRendering || "auto";
     const candleX = x - this.candleWidth / 2;
     return assign({
-      x: candleX, y, style: this.style, role, width: this.candleWidth, height: candleHeight,
-      shapeRendering, className
+      x: candleX,
+      y: Math.min(open, close),
+      style: this.style,
+      role,
+      width: this.candleWidth,
+      height: candleHeight,
+      shapeRendering,
+      className
     }, events);
   }
 
-  getWickProps(props) {
-    const { x, y1, y2, events, className } = props;
+  getWickProps(props, wickType) {
+    const { x, high, low, open, close, wickStrokeWidth, events, className } = props;
     const shapeRendering = props.shapeRendering || "auto";
     const role = props.role || "presentation";
-    return assign(
-      { x1: x, x2: x, y1, y2, style: this.style, role, shapeRendering, className },
-      events
-    );
+    const lowWick = Math.min(close, open);
+    const highWick = Math.max(close, open);
+    const wickStyle = defaults({}, this.style, { strokeWidth: wickStrokeWidth });
```

Comment:
This one is my fault, but the ordering for `defaults` is backwards. It should be `defaults({ strokeWidth: wickStrokeWidth }, this.style)`. Otherwise, the style will always override the `wickStrokeWidth`